src/core_plugins/kbn_doc_views/public/views/table.html

```diff
@@ -36,6 +38,7 @@
         <span ng-if="canToggleColumns()">
           <button
             class="doc-viewer-button"
+            aria-label="Toggle column in table"
```

Comment:
Since this doesn't have an `aria-expanded` attribute on it, can we dynamically update this label to be "Add column to table" or "Remove column from table" so the screen reader user can anticipate what will happen when they click it?

Reply:
Good catch. I actually added an `aria-pressed` instead, to indicate the state, whether or not this is turned on or off (usually used for toggle buttons).